### DISCUSSION TITLE
第六章_PyTorch模型定义与进阶训练技巧中的Carvana数据集下载？

### DISCUSSION THREAD

**Discussion by roki1031** [datawhalechina/thorough-pytorch#20] [Q&A] [answered]
rt

**Comment by roki1031** [comment, marked as answer]
下载地址：https://www.kaggle.com/competitions/carvana-image-masking-challenge
完整的数据集很大，我们的课程里面只需要用到黑白的mask图。
对于本课程，需要下载的数据有：train.zip, train_mask.zip

**Reply by ZhikangNiu** [reply]
你的回答对于我们的教程很有帮助！

**Reply by clorisqiu1** [reply]
是train_masks.zip
